src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerSearchViewModel.cs

```diff
@@ -895,7 +897,13 @@ internal IEnumerable<PackageManagerSearchElementViewModel> Search(string query)
                         PackageManagerClientViewModel.AuthenticationManager.HasAuthProvider,
                         CanInstallPackage(x.Name), isEnabledForInstall)))
                     .ToList();
+
                 Sort(list, this.SortingKey);
+
+                if (SortingDirection == PackageSortingDirection.Descending)
+                {
+                    list.Reverse();
+                }
```

Comment:
Maybe this can be part of the Sort function? So other functions would get the reverse behavior for free?

Reply:
That is already being done inside the Sort(), which will sort the SearchResults in the View. This case is when the package search window opens and the complete list of packages are showed. We want this list in descending order.

Reply:
Sorry for the confusion.. I mean do we need to make this part of the function `private static void Sort(List<PackageManagerSearchElementViewModel> results, PackageSortingKey key)`? The one we are calling above. Otherwise, we have two sorting function behaving differently, one respect `PackageSortingDirection` but the other do not

Reply:
@reddyashish I see. If you prefer that way, I am fine with it. Would you update the function description to reflect that just so ppl know when calling `sort()` in the future?

Reply:
Sort() is sorting the results by key and then by sorting direction. I have updated the function description.